extensions/esa-search/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "esa-search",
+  "title": "esa search",
+  "description": "Search for posts by your team's esa. No official esa.io product.",
+  "icon": "icon.png",
+  "author": "kawamataryo",
+  "categories": [
+    "Productivity",
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "searchPost",
+      "title": "Search esa Post",
+      "description": "Search for esa post.",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "keyword",
+          "placeholder": "keyword",
+          "type": "text",
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "createPost",
+      "title": "Create esa Post",
+      "description": "Create post for esa.",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "title": "default show detail",
+      "name": "isShowDetail",
+      "description": "Show detail screen in initial view",
```

Comment:
```suggestion
      "description": "	Show Detail Screen in Initial View",
```

Reply:
As far as the other commands are concerned, the `description` field is lower case, should this be changed?

Reply:
My bad, wrong line 🙂